Resources/Prototypes/_LostParadise/Reagents/Consumable/Drink/alcohol.yml

```
@@ -0,0 +1,45 @@
+  flavor: lppmiruvor
+  color: "#fcff54"
+  metamorphicSprite:
+    sprite: _LostParadise/Objects/Consumable/Drinks/miruvor.rsi
+    state: icon_empty
+  metamorphicMaxFillLevels: 5
+  metamorphicFillBaseName: fill-
+  metamorphicChangeColor: false
+  metabolisms:
+    Drink:
+      effects:
+      - !type:SatiateThirst
```

yamllint

[error] wrong indentation: expected 8 but found 6

(indentation)